modules/administration-guide/partials/con_architecture-overview-with-che-server.adoc

```diff
@@ -0,0 +1,25 @@
+[id="{prod-id-short}-architecture-overview-with-che-server_{context}"]
+= {prod-short} architecture with {prod-id-short}-server
+
+{prod-short} server is the default workspace engine.
+
+.High-level {prod-short} architecture with the {prod-short} server engine
+image::administration-guide:architecture/{project-context}-architecture-with-che-server-engine.png[]
+
+{prod} components are:
+
+Workspace controller::
+
+An always-running service managing user workspaces with the {orch-name} API.
```

Comment:
```suggestion
An always-running service that manages user workspaces with the {orch-name} API.
```

Reply:
"that" would not be a good fit here because:

Use "that", without a comma, to introduce a restrictive clause. Use "which", preceded by a comma, to introduce a nonrestrictive clause. (IBM SG)

Reply:
@themr0c This seems like a pretty good example of [a restrictive clause](https://www.lavc.edu/getattachment/writingcenter/Handouts/Clauses-III_-Restrictive_Nonrestrictive-ADA-2018.pdf?lang=en-US#:~:text=Restrictive%20clauses%20require%20no%20punctuation,them%20from%20the%20complete%20sentence.). It introduces information that is necessary to the meaning of the sentence